megablocks/layers/router.py

```diff
@@ -56,7 +56,8 @@ def forward(self, x):
         scores = self.layer(x.view(-1, x.shape[-1])).softmax(dim=-1)
         expert_weights, expert_indices = self._top_k(scores)
         if self.args.moe_normalize_expert_weights:
-            expert_weights /= expert_weights.sum(dim=-1, keepdim=True)
+            expert_weights /= torch.norm(expert_weights, p=self.args.moe_normalize_expert_weights,
```

Comment:
nit: can we start a new line before the first arg instead of after the second?

Reply:
done!